PusherServer.Tests/AcceptanceTests/Trigger.cs

```diff
@@ -415,4 +415,40 @@ public async Task It_should_expose_a_multiple_event_ids_when_publishing_to_multi
             Assert.AreEqual(channels.Length, result.EventIds.Count);
         }
     }
+
+    [TestFixture]
+    public class When_requesting_channel_attributes
+    {
+        private static string TEST_CHANNEL_NAME = "test-channel-1";
+
+        [Test]
+        public async Task It_should_return_channel_attributes()
+        {
+            IPusher pusher = new Pusher(Config.AppId, Config.AppKey, Config.AppSecret, new PusherOptions()
+            {
+                HostName = Config.HttpHost,
+                BatchEventDataSizeLimit = PusherOptions.DEFAULT_BATCH_EVENT_DATA_SIZE_LIMIT,
+            });
+
+            var channels = new List<string>() { TEST_CHANNEL_NAME };
+            var options = new TriggerOptions
+            {
+                Info = new List<string>() { "subscription_count" },
+            };
+
+            var client = new PusherClient.Pusher(Config.AppKey, new PusherClient.PusherOptions
+            {
+                Cluster = Config.Cluster,
+                TraceLogger = new PusherClient.TraceLogger(),
+            });
+            await client.ConnectAsync().ConfigureAwait(false);
+            var channel = await client.SubscribeAsync(TEST_CHANNEL_NAME).ConfigureAwait(false);
+            channel.Bind("event", delegate (PusherClient.PusherEvent data) { });
+
+            var result = await pusher.TriggerAsync(channels.ToArray(), "event", "data", options).ConfigureAwait(false);
+
+            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
+            Assert.AreEqual(1, result.ChannelAttributes[TEST_CHANNEL_NAME].subscription_count);
+        }
+    }
```

Comment:
We can simplify the test using some of the factories that I found already exist (thanks to the previous developer 🙂 ). 

I also added two more tests in the following code. Could you please include them in your PR? (I don't have access to your fork)
```c#
    [TestFixture]
    public class When_requesting_channel_attributes
    {
        [Test]
        public async Task It_should_return_subscription_count_for_any_channel()
        {
            const string channelName = "test-channel-1";
            var pusher = ClientServerFactory.CreateServer();
            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);

            var options = new TriggerOptions
            {
                Info = new List<string>() { "subscription_count" },
            };
            var result = await pusher.TriggerAsync(channelName, "event", "data", options).ConfigureAwait(false);

            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
            Assert.AreEqual(1, result.ChannelAttributes[channelName].subscription_count);
        }
        
        [Test]
        public async Task It_should_return_user_count_for_presence_channels()
        {
            const string channelName = "presence-test-channel-1";
            var pusher = ClientServerFactory.CreateServer();
            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);

            var options = new TriggerOptions
            {
                Info = new List<string>() { "user_count" },
            };
            var result = await pusher.TriggerAsync(channelName, "event", "data", options).ConfigureAwait(false);

            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
            Assert.AreEqual(1, result.ChannelAttributes[channelName].user_count);
        }
    
        [Test]
        public async Task It_should_return_all_requested_attributes()
        {
            const string channelName = "presence-test-channel-2";
            var pusher = ClientServerFactory.CreateServer();
            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);
            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);
            await ClientServerFactory.CreateClientAsync(pusher, channelName).ConfigureAwait(false);

            var options = new TriggerOptions
            {
                Info = new List<string>() { "subscription_count", "user_count" },
            };
            var result = await pusher.TriggerAsync(channelName, "event", "data", options).ConfigureAwait(false);

            Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
            Assert.AreEqual(1, result.ChannelAttributes[channelName].user_count);
            Assert.AreEqual(3, result.ChannelAttributes[channelName].subscription_count);
        }
    }
```

Reply:
I've simplified my test and added yours. 

I've also added a function to generate random channels names to prevent issues with subscription/user counts. If you prefer that I revert to static channels names, please, let me know.